Gemfile

```diff
@@ -28,14 +28,14 @@ group :assets do
   gem 'execjs'        , '~> 2.0.0' # Will use system-available JS runtime
 end
 
-group :development, :test do
-  gem 'mocha', '~> 0.13.3', :require => false
-  gem 'sqlite3'
-  gem 'rspec-rails', '~> 2.13.0'
-  gem 'factory_girl', '< 3.0' # supports ruby 1.8.7
-  gem 'shoulda-matchers', '< 2.0' # support ruby 1.8.7
-  gem 'rspec-html-matchers'
-  gem 'nokogiri', '< 1.6' # support ruby 1.8.7
+gem :development, :test do
```

Comment:
Oh, it's `group` not `gem`. Would be best to change this block to exactly match the prior Gemfile contents.